bitnami/cassandra/README.md

```diff
@@ -46,81 +46,87 @@ The command removes all the Kubernetes components associated with the chart and
-| `networkPolicy.enabled`              | Enable NetworkPolicy                                                                                                                                | `false`                                                      |
-| `networkPolicy.allowExternal`        | Don't require client label for connections                                                                                                          | `true`                                                       |
-| `metrics.enabled`                    | Start a side-car prometheus exporter                                                                                                                | `false`                                                      |
-| `metrics.image.registry`             | Cassandra exporter Image registry                                                                                                                   | `docker.io`                                                  |
-| `metrics.image.repository`           | Cassandra exporter Image name                                                                                                                       | `criteo/cassandra_exporter`                                  |
-| `metrics.image.tag`                  | Cassandra exporter Image tag                                                                                                                        | `2.0.4`                                                      |
-| `metrics.image.pullPolicy`           | Image pull policy                                                                                                                                   | `IfNotPresent`                                               |
-| `metrics.image.pullSecrets`          | Specify docker-registry secret names as an array                                                                                                    | `[]` (does not add image pull secrets to deployed pods)      |
-| `metrics.podAnnotations`             | Additional annotations for Metrics exporter                                                                                                         | `{prometheus.io/scrape: "true", prometheus.io/port: "8080"}` |
-| `metrics.resources`                  | Exporter resource requests/limit                                                                                                                    | `{}`                                                         |
+| Parameter                            | Description                                                                                                                                               | Default                                                      |
```

Comment:
I would add a section explaining the Volume Permissions init container. 

In addition, in the NOTES I will add a section where, in case the volume container is not enabled, let the user know that, in some platforms, it may be necessary enabling it.

Reply:
I agree with the README section but not sure about the section in the NOTES. By default, the volumePermission initContainer will be disabled and it is supposed to be used only with a minority of kubernetes versions. 
My concern is that we will add "noise" for user that are not affected by the issue and the could be confused.

Reply:
Ok, then just the README